vouches/src/lib.rs

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2019 Encointer Association
+// This file is part of Encointer
+//
+// Encointer is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Encointer is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Encointer.  If not, see <http://www.gnu.org/licenses/>.
+
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use core::marker::PhantomData;
+use encointer_primitives::vouches::{Vouch, VouchQuality, VouchType};
+use frame_system::{self as frame_system, ensure_signed, pallet_prelude::OriginFor};
+use log::info;
+pub use pallet::*;
+use sp_std::convert::TryInto;
+pub use weights::WeightInfo;
+// Logger target
+const LOG: &str = "encointer::vouches";
+
+#[cfg(feature = "runtime-benchmarks")]
```

Comment:
I would like this to be `cfg(any(test, feature = "runtime-benchmarks"))` then we can at least test if the benchmarks work in std.

Reply:
introducing this in this PR would be inconsistent with other pallets. We should have a separate issue if we want to change that behaviour